### DISCUSSION TITLE
umi4(非max)使用dva，插件引入，开启配置启动项目会报错

### DISCUSSION THREAD

**Discussion by Santaizii** [umijs/umi#11472] [Q&A]
![屏幕截图 2023-07-29 181223](https://github.com/umijs/umi/assets/131859590/ac690451-3aa5-46fd-b88c-821895e7c2d2)
![屏幕截图 2023-07-29 181325](https://github.com/umijs/umi/assets/131859590/61366b2a-d8e3-46c7-b6fa-3297032f64ef)
![屏幕截图 2023-07-29 181346](https://github.com/umijs/umi/assets/131859590/2efca037-832b-443b-b563-7c42a61926a3)

**Comment by fz6m** [comment]
你装 `@umijs/plugins` 这个包了吗，把这个包和 `umi` 都更新到最新版本试试。

如还有问题，需要给一个最小复现。

**Reply by Santaizii** [reply]
![屏幕截图 2023-07-29 232459](https://github.com/umijs/umi/assets/131859590/842f0c5f-1f3c-4113-8107-cb8248034d71)

**Reply by Santaizii** [reply]
有这个包的

**Reply by Santaizii** [reply]
谢谢，解决了，umi更新到最新
